docs: add migration guide, api reference for network mode (#1501)
toss/suspensive: docs/suspensive.org/src/content/en/docs/react-query/infiniteQueryOptions.mdx

```

</Sandpack>

### Version History

| Version  | Changes                                                                                                                                                                           |
| -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | `v3.0.0` | The default value of `networkMode` in `@suspensive/react-query` has been fixed to `'always'`. For more details, please refer to the [Migration to v3 guide](./migrate-to-v3.mdx). |

Reviewer: ```suggestion
| `v3.0.0` | `networkMode` has been removed. For more details, please refer to the [Migration to v3 guide](./migrate-to-v3.mdx). |
```
